2024-10-09 23:32:10,232 INFO    MainThread:545047 [wandb_setup.py:_flush():79] Current SDK version is 0.18.3
2024-10-09 23:32:10,232 INFO    MainThread:545047 [wandb_setup.py:_flush():79] Configure stats pid to 545047
2024-10-09 23:32:10,232 INFO    MainThread:545047 [wandb_setup.py:_flush():79] Loading settings from /home/autrio/.config/wandb/settings
2024-10-09 23:32:10,232 INFO    MainThread:545047 [wandb_setup.py:_flush():79] Loading settings from /home/autrio/college-linx/SMAI/smai-m24-assignments-Autrio/wandb/settings
2024-10-09 23:32:10,232 INFO    MainThread:545047 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2024-10-09 23:32:10,232 INFO    MainThread:545047 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-10-09 23:32:10,232 WARNING MainThread:545047 [wandb_setup.py:_flush():79] Could not find program at -m assignments.3.2.2.2
2024-10-09 23:32:10,232 INFO    MainThread:545047 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program_relpath': None, 'program': '-m assignments.3.2.2.2'}
2024-10-09 23:32:10,232 INFO    MainThread:545047 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-10-09 23:32:10,232 INFO    MainThread:545047 [wandb_init.py:_log_setup():532] Logging user logs to /home/autrio/college-linx/SMAI/smai-m24-assignments-Autrio/wandb/run-20241009_233210-agk7266t/logs/debug.log
2024-10-09 23:32:10,232 INFO    MainThread:545047 [wandb_init.py:_log_setup():533] Logging internal logs to /home/autrio/college-linx/SMAI/smai-m24-assignments-Autrio/wandb/run-20241009_233210-agk7266t/logs/debug-internal.log
2024-10-09 23:32:10,232 INFO    MainThread:545047 [wandb_init.py:init():617] calling init triggers
2024-10-09 23:32:10,232 INFO    MainThread:545047 [wandb_init.py:init():624] wandb.init called with sweep_config: {}
config: {'lr': 0.001, 'batch_size': 256, 'epoch': 10000, 'optimizer': 'mini-batch', 'loss_fn': <models.MLP.MLP2.loss.MSELoss object at 0x7b7cd4721690>, 'activation': <models.MLP.MLP2.activations.Relu object at 0x7b7cd46d8040>, 'type': 'classification'}
2024-10-09 23:32:10,232 INFO    MainThread:545047 [wandb_init.py:init():667] starting backend
2024-10-09 23:32:10,232 INFO    MainThread:545047 [wandb_init.py:init():671] sending inform_init request
2024-10-09 23:32:10,233 INFO    MainThread:545047 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-10-09 23:32:10,233 INFO    MainThread:545047 [wandb_init.py:init():684] backend started and connected
2024-10-09 23:32:10,234 INFO    MainThread:545047 [wandb_init.py:init():779] updated telemetry
2024-10-09 23:32:10,237 INFO    MainThread:545047 [wandb_init.py:init():812] communicating run to backend with 90.0 second timeout
2024-10-09 23:32:10,786 INFO    MainThread:545047 [wandb_init.py:init():863] starting run threads in backend
2024-10-09 23:32:10,988 INFO    MainThread:545047 [wandb_run.py:_console_start():2465] atexit reg
2024-10-09 23:32:10,988 INFO    MainThread:545047 [wandb_run.py:_redirect():2313] redirect: wrap_raw
2024-10-09 23:32:10,989 INFO    MainThread:545047 [wandb_run.py:_redirect():2378] Wrapping output streams.
2024-10-09 23:32:10,989 INFO    MainThread:545047 [wandb_run.py:_redirect():2403] Redirects installed.
2024-10-09 23:32:10,990 INFO    MainThread:545047 [wandb_init.py:init():907] run started, returning control to user process
2024-10-09 23:32:59,477 WARNING MsgRouterThr:545047 [router.py:message_loop():77] message_loop has been closed
